content/blog/2025-11-22-napari-plugin-sustainability-intro.md

```diff
@@ -0,0 +1,122 @@
+---
+
+Foundational scientific software tools deliver innovative, yet reliable methods for research across multiple domains.
+These tools often rely on community expansion by "downstream" libraries to meet domain-specific needs. [napari](https://napari.org/), an open-source, multidimensional imagee viewer for Python, provides core infrastructure for visualizing and annotating scientific imaging data. Its extendable GUI has empowered a diverse ecosystem of over 550 [community-developed plugins](https://napari-hub.org/) (i.e., downstream libraries). Since the public announcement of napari in 2019 and the addition of plugin support soon after, collaboration among core contributors, plugin authors, and users has enabled research workflows that no single library could accomplish alone.
+
+One advantage of napari and its plugin ecosystem is that it enables advanced scientific processing, without necessarily requiring any code! This lowers the barrier to entry for domain scientists who may be beginners to writing code, but need powerful tools to analyze their data. Then, as users meet the limits of available resources, they can learn to extend napari and their image analysis with their own code. In this way, napari serves as an *entry point* into scientific programming for many researchers, just like other foundational scientific software tools.
+
+The napari project maintains key infrastructure for plugin developers and users, including an easily searchable [napari-hub](https://napari-hub.org/) website, a GUI to easily download plugins with the [napari-plugin-manager](https://github.com/napari/napari-plugin-manager), and tools to easily create plugins (including [documentation](https://napari.org/stable/plugins/index.html) and the [napari-plugin-template](http://github.com/napari/napari-plugin-template)). In recent years, collaboration between plugin developers and the napari core team has declined. This has led to challenges in maintaining a healthy plugin ecosystem, with many developers struggling to keep their plugins up to date and aligned with napari's evolving architecture. This sustainability problem is not unique to napari; its a pattern repeated across scientific software, where the relationship between foundational tools and their domain-specific extensions takes on a transactional nature, rather than a collaborative one.
+(You can read more about sustainability efforts in core napari at this [recent blog post](https://napari.org/island-dispatch/blog/sustainability_survey.html).)
+
+With financial support from the [United States Research Software Sustainability Institute (URSSI) Early Career Fellowship](https://urssi.us/blog/2025/07/02/call-for-proposals-urssi-early-career-fellows-round-2/), I've been working with the napari community to improve the plugin ecosystem's sustainability. Our goal is to build infrastructure and practices that other scientific software communities can learn from and adapt to meet their own needs.
```

Comment:
Normally we just spell out URSSI as "US Research Software Sustainability Institute"; also, I would use a separate link on the URSSI name to https://urssi.us and then just link Early Career Fellowship to the relevant blog post (in part to avoid the issues with links not breaking in hugo)